packages/ui-kit/src/components/code-block.tsx

```diff
@@ -181,7 +187,7 @@ const getLineStyles = (theme, options) => {
  * yarn start
  * ```
  */
-const languageAliases = {
+const languageAliases: { [key: string]: Language } = {
```

Comment:
Nit: we can also use `Record`

```suggestion
const languageAliases: Record<string, Language> = {
```

Reply:
I had the impression that the index form is conventional when any string is allowed and Record is interesting when you e.g. have a specific union or so as keys.